packages/chat/tests/messageUtils.test.ts

```diff
@@ -0,0 +1,110 @@
+import { autoSplitterWithSpaces } from '../src/utils/messageUtil';
+
+describe('Auto Text Splitter with Spaces', () => {
```

Comment:
This file as a whole needs to be within the `src` folder, in a folder named `__tests` next to the file being tested, with the same filename but ending in `.test.ts`. So in this case it should be `packages/chat/src/utils/__tests__/messageUtil.test.ts`